packages/peregrine/lib/talons/Wishlist/WishlistDialog/useWishlistDialog.js

```diff
@@ -72,7 +80,7 @@ export const useWishlistDialog = props => {
         handleCancel,
         handleCancelNewList,
         handleNewListClick,
-        isAddLoading,
+        isLoading: isLoading || isAddLoading,
```

Comment:
Note for myself: Gotta check if I can make this a bit more explicit.

Reply:
@revanth0212 is this still relevant i also don't have a better solution

Reply:
Yeah dude, ill have to work and update this piece of code.